### DISCUSSION TITLE
What about more database possibilities?

### DISCUSSION THREAD

**Discussion by Olyno** [payloadcms/payload#106] [Feature Requests & Ideas]
Hi 👋🏻 
So, I started to have a look in payloadcms to see how it works, and the first question I got was: why can't we use another database than MongoDB?

MongoDB is great for a lot of things, but it makes the databases really big and not scalable (in my opinion). Implementing another database as postgres could be great! We can do it using [TypeORM](https://typeorm.io/#/) which experiment MongoDB too.

What do you think about it?

**Comment by DanRibbens** [comment]
Hi @Olyno,
Thanks for checking out Payload and giving feedback! 

We've talked in depth about implementing other databases. During development we analyzed the ORMs available and decided to push forward on MongoDB. That decision was the best choice for us to make our initial release and get developer feedback. Since release we have been working away from relying on Mongoose plugins for some of the core features, such as localization. In that regard we are closer now than ever before to offering other DB support. There is still a substatial amount of work to adopt a new ORM though.

It will happen, its just matter of when. We will announce a roadmap to add clarity soon. I'll keep TypeORM in mind for when we plan the feature.
Thanks!

**Reply by supermx1** [reply]
Hey guys, great work. Just started using Payload and I am interested in integrating with other databases and MongoDB is not convenient for many of clients that prefer having their data managed entirely by them.

**Reply by supermx1** [reply]
You can consider Prisma as the ORM

**Reply by carlosbensant** [reply]
+1 for Prisma which also offers MongoDB support.

**Comment by owieth** [comment]
Most recent activities regarding database possibilities: #287
